open-grant-proposals/open-proposal-blockchain-package-manager.md

```diff
@@ -0,0 +1,183 @@
+# Open Grant Proposal: `Blockchain Package Manager - FileCoin Package`
```

Comment:
This sounds very useful. Do you think this is the right time to integrate with Filecoin, or would it make sense to wait until we launch our testnet and the protocol, API & implementations stabilize?